vespa-hadoop/pom.xml

```diff
@@ -145,8 +155,28 @@
             </plugin>
 
             <plugin>
-                <groupId>com.yahoo.vespa</groupId>
-                <artifactId>abi-check-plugin</artifactId>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.0</version>
+                <configuration>
+                    <jdkToolchain>
+                        <version>${java.version}</version>
+                    </jdkToolchain>
+                    <source>${java.version}</source>
+                    <target>${java.version}</target>
+                    <showWarnings>true</showWarnings>
+                    <optimize>true</optimize>
+                    <showDeprecation>false</showDeprecation>
+                    <compilerArgs>
```

Comment:
AFAICS these are identical to those in 'parent' and can just be inherited from there.  Just remove the whole `compilerArgs`. The same goes for the three options above (showWarnings, optimize, showDeprecation).